From e1bc987c482bd8f0eadae88a78d3fc91c1c27de3 Mon Sep 17 00:00:00 2001
From: Andi Kleen <ak@linux.intel.com>
Date: Thu, 1 Aug 2024 01:30:40 -0700
Subject: [PATCH 20/32] Add missing hunks from last change

---
 common/os/os_win.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/common/os/os_win.c b/common/os/os_win.c
index 48052c6..9aaefae 100644
--- a/common/os/os_win.c
+++ b/common/os/os_win.c
@@ -139,7 +139,6 @@ node_cpu_string(node_t *node, char *s1, int size)
 	int i, j, k, l, cpuid_start;
 	int *cpuid_arr;
 	int ncpus;
-	int s1_len = size;
 	perf_cpu_t *cpus = node_cpus(node);
 
 	s1[0] = 0;
@@ -174,8 +173,6 @@ node_cpu_string(node_t *node, char *s1, int size)
 	for (j = 1; j < ncpus; j++) {
 		k++;
 		if (cpuid_arr[j] != cpuid_start + l) {
-			int s2_len = sizeof(s2);
-
 			if (k < ncpus) {
 				if (l == 1) {
 					print_buf(&s1, &size, "%d ", cpuid_start);
@@ -190,8 +187,6 @@ node_cpu_string(node_t *node, char *s1, int size)
 					print_buf(&s1, &size, "%d-%d",
 						cpuid_start, cpuid_start + l - 1);
 				}
-				s2_len -= strlen(s2);
-
 				print_buf(&s1, &size, " %d", cpuid_arr[j]);
 			}
 
-- 
2.41.0

